Source/Bogus/DataSets/Lorem.cs

```diff
@@ -24,53 +26,61 @@ public string Word()
       }
 
       /// <summary>
-      /// Get some lorem words
+      /// Get a number of random lorem words
       /// </summary>
+      /// <param name="num">
+      /// The number of random lorem words to be gotten.
+      /// </param>
       public string[] Words(int num = 3)
       {
-         return Enumerable.Range(1, num).Select(f => Word()).ToArray(); // lol
+         return Enumerable.Range(1, num).Select(_ => Word()).ToArray();
       }
 
       /// <summary>
       /// Get a character letter.
       /// </summary>
-      /// <param name="num">Number of characters to return.</param>
+      /// <param name="num">The number of characters to return.</param>
       public string Letter(int num = 1)
       {
          if( num <= 0 )
             return string.Empty;
 
-         var w = Words(1)[0];
-         var c = Random.ArrayElement(w.ToArray());
-         return c + Letter(num - 1);
+         var words = Words(1)[0];
+         var characters = Random.ArrayElement(words.ToArray());
+         return characters + Letter(num - 1);
       }
 
       /// <summary>
       /// Get a random sentence of specific number of words. 
       /// </summary>
-      /// <param name="wordCount">Get a sentence with wordCount words. Defaults between 3 and 10</param>
-      /// <param name="range">Add anywhere between 0 to 'range' additional words to wordCount. Default is 0.</param>
+      /// <param name="wordCount">
+      /// Get a sentence with wordCount words. Defaults between 3 and 10
+      /// </param>
+      /// <param name="range">
+      /// Add anywhere between 0 to 'range' additional words to wordCount. Default is 0.
+      /// </param>
       public string Sentence(int? wordCount = null, int? range = 0)
       {
-         var wc = wordCount ?? this.Random.Number(3, 10);
+         wordCount = wordCount ?? this.Random.Number(3, 10);
```

Comment:
I'm not sure about overwriting the parameter values out-right here. Sometimes it's okay, sometimes not. Also, some of these methods are almost 1-1 with faker.js; which I'd prefer for maintenance reasons keep some of those variable names. In case there's ever an update to one of these methods, porting/updating these methods with same variable names helps ease the process. When we go about changing variable names like this it can introduce some friction for me to rebuild the program state in my head to get a picture of what variable holds what state and makes the maintenance/porting process more difficult. I don't know if this is one of those methods, but when possible, try to avoid renaming variables if the variables are the same used in faker.js. If there's is something more fundamental about the algorithm that can be more idiomatic in C#, then we can discuss changing it.

Reply:
Understandable, didn't know this was more of a direct port. Reverted these instances.